package.json

```diff
@@ -152,9 +152,9 @@
     "@types/pako": "^1.0.1",
     "@types/parse5": "^6.0.0",
     "@types/qrcode": "^1.3.5",
-    "@types/react": "17.0.14",
+    "@types/react": "^18.0.0",
     "@types/react-beautiful-dnd": "^13.0.0",
-    "@types/react-dom": "17.0.9",
+    "@types/react-dom": "^18.0.0",
     "@types/react-test-renderer": "^17.0.1",
```

Comment:
This probably needs updating too

We also need a react 18 enzyme adapter, see L164 below

Right now you have a conflicting version of react-dom being imported by something

```


node_modules/@types/react-dom/node_modules/@types/react/index.d.ts:3057:14 - error TS2300: Duplicate identifier 'LibraryManagedAttributes'.
--
  |  
  | 3057         type LibraryManagedAttributes<C, P> = C extends React.MemoExoticComponent<infer T> \| React.LazyExoticComponent<infer T>
  | ~~~~~~~~~~~~~~~~~~~~~~~~
  |  
  | node_modules/@types/react/index.d.ts:3127:14
  | 3127         type LibraryManagedAttributes<C, P> = C extends React.MemoExoticComponent<infer T> \| React.LazyExoticComponent<infer T>
  | ~~~~~~~~~~~~~~~~~~~~~~~~
  | 'LibraryManagedAttributes' was also declared here.

Reply:
On L164 :   `"@wojtekmaj/enzyme-adapter-react-17": "^0.6.1"`
The Author of this package has published a [Blog](https://dev.to/wojtekmaj/enzyme-is-dead-now-what-ekl) saying that there won't be an enzyme adapter for React 18. I searched for other enzyme adapters as well but couldn't find any.

The `error TS2300: Duplicate identifier 'LibraryManagedAttributes'` would probably be resolved now after I updated `"resolutions": {
    "@types/react": "^18.0.0"
  }`  as per this stack overflow [answer](https://stackoverflow.com/questions/52399839/duplicate-identifier-librarymanagedattributes). 

Also updated `"@types/react-test-renderer"` which had react 17 in its dependencies.

Reply:
> The Author of this package has published a [Blog](https://dev.to/wojtekmaj/enzyme-is-dead-now-what-ekl) saying that there won't be an enzyme adapter for React 18. I searched for other enzyme adapters as well but couldn't find any.

Then we would need to move away from Enzyme to be able to upgrade to React 18?

Reply:
It is the longer-term plan to move away from Enzyme to RTL but it'd suck if we have to block the 18 upgrade on it